### ISSUE TITLE
paddleocr outputs everything as image

### ISSUE THREAD

**Issue by DOOMBOT0** [PaddlePaddle/PaddleOCR#17387]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

i trained devanagari paddleocr v5 mobile text recognition module, now when i use the trained model i get the input image as output without any text recognition done, the output text field is empty, i even tried it on the data used in training, the accuracy it gives is 0.9999something

### 🏃‍♂️ Environment (运行环境)

os fedora 42
podman
using paddlex container

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

drive link to the trained module: https://drive.google.com/drive/folders/1PFS6wMD8--16hUhC0bVuVw9ksvVBfmju?usp=sharing

**Comment by DOOMBOT0** [comment]
@changdazhou, what could be the cause of the issue?

**Comment by scyyh11** [comment]
Did you specify the training dictionary during inference?

**Comment by zhangyubo0722** [comment]
Please provide the command to export the inference model, as well as a test image to verify that the model is loaded correctly.

**Comment by DOOMBOT0** [comment]
> Did you specify the training dictionary during inference?

@scyyh11 i did not specify the training dictionary during inference, but i did use the config file within the trained model folder, the config file contains the characters added before starting training.
Thank you for taking your time to reply.

**Comment by DOOMBOT0** [comment]
> Please provide the command to export the inference model, as well as a test image to verify that the model is loaded correctly.

@zhangyubo0722 i did not export the inference model, it was already there, i am using the paddlex container and paddlex automatically produced an inference folder with the .pdiparams, .json, .yml file.
 What kind of image should i provide, the ones i tried inference or perhaps the terminal logs when passing the predict command?
Here is one of the images i tried the model on, if that is what you wanted:

<img width="571" height="131" alt="Image" src="https://github.com/user-attachments/assets/06692083-3f23-415a-942d-8f76f92243e4" />

Please tell me if you need to see anything else. 
Thank you for taking your time to reply.

**Comment by DOOMBOT0** [comment]
i have tried both terminal command and the python script

CLI command: python main.py   -c ./output/best_accuracy/inference/inference.yml   -o Global.mode=predict   -o Predict.model_dir="./output/best_accuracy/inference"   -o Predict.input="./paddle/Dataset/Xtraining/muna.pdf"

python script: 
from paddlex import create_model
model = create_model(model_name="devanagari_PP-OCRv5_mobile_rec",model_dir="./output/best_accuracy/inference")
output = model.predict("./paddle/Dataset/Xtraining/01.png", batch_size=1)
for res in output:
    res.print(json_format=False)
    res.save_to_img("./paddle/output/")
    res.save_to_json("./paddle/output/res.json")

Both of them gave the same result.

**Comment by DOOMBOT0** [comment]
@scyyh11 @zhangyubo0722 @changdazhou  i tried training the module in a container with only paddlepaddle and paddleocr installed, i got the same result
this time i performed the training on just 60 images. I have been using the devanagari_PP-OCRv5_mobile_rec_pretrained.pdparams model to perform training up till now, maybe there is a problem with the pretrained model. I followed the instructions from here https://www.paddleocr.ai/main/en/version3.x/module_usage/text_recognition.html#42-model-trainingl.
here is the result of training: 
https://drive.google.com/drive/folders/1no7ZNIgM8gCOnJ1B7azEJ1UqYXqSn3E6?usp=sharing
My friend also tried to train the same model in a colab environment using paddlepaddle  on about 80,000 images and got the same result.

**Comment by DOOMBOT0** [comment]
@zhangyubo0722 @scyyh11 @changdazhou i trained the devanagari mobile v3 model on just 60 images,  the training was successful

<img width="1887" height="1015" alt="Image" src="https://github.com/user-attachments/assets/04f02ecc-d13f-4ddb-8fc1-5080acee807d" />
the leftmost image is the output of the model i trained, the middle one is the output of the devanagari v5 mobile model, and the rightmost image is the original image. Both the v3 and v5 models were used inside the ppstruct pipeline. Now, the problem is that the model i trained is not giving spaces between words.  Will that improve with further training? 
The data i am training on consists of only single character and single word images, could that be the reason for the absence of spaces in the output?
